GOVERNANCE.md

```diff
@@ -0,0 +1,67 @@
+
+## Projects
+
+Each project must have a MAINTAINERS.md file with at least one maintainer. Where a project has a release process, access and documentation should be such that more than one person can perform a release. Releases should be announced on the prometheus-users mailing list. Any new repositories should be first proposed on prometheus-team@ and following the voting procedures listed below. When a repository is no longer relevant, it should be moved to the prometheus-junkyard Github organization.
+
+## Committers and Maintainers
+
+Committer status (commit bit) is given to those who have made ongoing contributions to a given project. This is usually in the form of code improvements, notable work on documentation, organizing events or user support could also be taken into account. 
+
+Maintainers are committers who are expected to lead the project and serve as a point of conflict resolution amongst the committers.
+
+In order to become a committer or maintainer, an existing maintainer must nominate you on the public prometheus-developers@ mailing list. Once another maintainer (or core developer) seconds and there are no objections within a week, a core developer will add the nominee to the prometheus-team@ mailing list and give appropriate other access rights. If there is an objection which can't be resolved via consensus, a formal [vote](#heading=h.x907eu6orfvp) will occur. A maintainer or committer may resign by notifying the prometheus-team@ mailing list. A maintainer with no project activity for a year will be deemed to have resigned.
+
+## Voting
+
+The Prometheus project usually runs by informal consensus, however sometimes a formal decision must be made. All votes MUST be done on the prometheus-developers@ mailing list, and generally at least a week given for discussion between maintainers and for votes to take place. A majority of Core Developers must vote in favor for a vote to pass.
+
+Formal votes must happen for the following: major changes to this document or the project structure, financial matters, adding core developers, removing committers or core developers.
+
+## Releases
+
+The release process hopes to encourage early, consistent consensus-building during project development. The mechanisms used are regular community communication on the mailing list about progress, scheduled meetings for issue resolution and release triage, and regularly paced and communicated releases. Each project within Prometheus can define their own release process if there’s consensus amongst the maintainers.
+
+## FAQ
+
+**How do I propose a vote?**
+
+Send an email to prometheus-dev with your motion. A majority of Core Developers must vote in favor (+1 or LGTM) for a vote to pass.
```

Comment:
`prometheus-dev` -> `prometheus-developers@` (the `@` to be consistent with the rest of email group mentions in this document, though arguably we should expand them all to `@googlegroups.com` to make it clear to others what these lists are).